lib/trino-hdfs/src/main/java/io/trino/hdfs/TrinoFileSystemCache.java

```diff
@@ -61,7 +61,7 @@ public class TrinoFileSystemCache
 
     public static final String CACHE_KEY = "fs.cache.credentials";
 
-    public static final TrinoFileSystemCache INSTANCE = new TrinoFileSystemCache();
+    static final TrinoFileSystemCache INSTANCE = new TrinoFileSystemCache();
```

Comment:
The commit message is "Remove Hadoop superclass for TrinoFileSystemCache stats", but I don't see a change like that in this commit.  Is the comment old?

Reply:
I'll add a commit description. The old code in `HiveModule` was doing `TrinoFileSystemCache.INSTANCE.getFileSystemCacheStats()` which required Hadoop to be on the compile time classpath since the superclass of `TrinoFileSystemCache` is `org.apache.hadoop.fs.FileSystemCache`. This changes how the stats instance is referenced to avoid the compile time reference.